pages/Product/components/VariantSelects/__snapshots__/spec.jsx.snap

```diff
@@ -323,184 +323,12 @@ exports[`<VariantSelects /> should render with variants 1`] = `
                       </span>
                     </button>
                   </ProductVariantPickerButton>
-                  <Component
+                  <Portal
```

Comment:
This snapshot shouldn't have been updated in that way. At the beginning of the test, a mock for the `react-portal` component is created, which just returns the passed children. This is why the output of the snapshot is that long.